decentraland/unity-explorer
# fix: change saturation slider color on hue change

## What does this PR change?


Fix #2258 

Colors the saturation slider based on which color is chosen through the hue slider

## How to test the changes?



1. Launch the explorer
2. Open the backpack
3. Select the body shape and click on color
4. Verify that the saturation slider color matches the avatar color
5. Verify that by changing the hue slider, the saturation slider color changes accordingly
6. Verify that by clicking on a color preset the saturation slider color changes accordingly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
